pkg/minikube/out/register/register.go

```diff
@@ -69,6 +69,7 @@ func init() {
 				SelectingDriver,
 				DownloadingArtifacts,
 				StartingNode,
+				PullingBaseImage,
```

Comment:
please also add a comment here
https://github.com/medyagh/minikube/blob/8fc54b430ba616dbadb129c62c20053d2e1d9390/pkg/minikube/out/register/register.go#L27
that each time a new step is added to consts it has to be added to this initial setup...

or if we could find a way to get list of the steps automatically from a list so we dont have to update this

Reply:
Done